src/services/logger.ts

```diff
@@ -4,21 +4,29 @@ import * as Sentry from '@sentry/react-native';
 interface LoggerOptions {
   stringifyJson?: boolean;
   emodjiPrefix?: '🟢' | '🔵' | '🟣' | '🔴' | '⚪️' | '🟡' | '🟠' | '🟤' | '⚫️';
+  // default true
```

Comment:
Maybe add this into JSDoc comment? Because this comment can be found only in this file...
```
/**
 *  @default enabled true
 */
 ```